src/components/dropdown/Dropdown1.jsx

```diff
@@ -0,0 +1,151 @@
+  }, []);
+
+  const handleSelect = (option) => {
+    setSelectedValue(option);
+    setIsOpen(false);
+    if (onSelect) {
+      onSelect(option);
+    }
+  };
+
+  return (
+    <Container ref={dropdownRef}>
+      <SelectButton 
+        $isOpen={isOpen}
+        className="h5-regular"
+      >
+        <TextWrapper>
+          <SelectedText>
+            {selectedValue || placeholder}
+          </SelectedText>
+          {isOpen && (
+            <OptionsList>
+              {options.map((option, index) => (
+                <OptionItem
+                  key={index}
+                  onClick={() => handleSelect(option)}
+                  $isSelected={selectedValue === option}
+                  className="h5-regular"
+                >
+                  {option}
+                </OptionItem>
+              ))}
+            </OptionsList>
+          )}
+        </TextWrapper>
+        <ArrowButton onClick={() => setIsOpen(!isOpen)} $isOpen={isOpen}>
+          <svg xmlns="http://www.w3.org/2000/svg" width="18" height="10" viewBox="0 0 18 10" fill="none">
+            <path d="M0.707092 0.707031L8.87376 8.8737L17.0404 0.707031" stroke="white" strokeWidth="2" strokeLinejoin="round"/>
+          </svg>
+        </ArrowButton>
+      </SelectButton>
+    </Container>
+  );
+};
+
+export default DropDown1;
+
+const Container = styled.div`
+  position: relative;
+  width: fit-content;
+`;
+
+const SelectButton = styled.button`
+  padding: 0;
+  display: flex;
+  align-items: center;
+  background: var(--common-100);
+  border: none;
+  cursor: default;
+`;
+
+const TextWrapper = styled.div`
+  position: relative;
+  flex: 1;
+`;
+
+const SelectedText = styled.span`
+  display: block;
+  padding: 1rem 1.5rem;
+  text-align: left;
+  color: var(--neutral-20);
+  border: 1px solid var(--neutral-90);
+`;
+
+const ArrowButton = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 1.5rem 1.2rem;
+  background: var(--neutral-20);
+
+  cursor: pointer;
+  transition: all 0.2s ease;
+
+  &:hover {
+    filter: brightness(0.9);
+  }
+
+  svg {
+    display: block;
+    transform: ${(props) => (props.$isOpen ? 'rotate(180deg)' : 'rotate(0deg)')};
+    transition: transform 0.2s ease;
+  }
+`;
+
+const OptionsList = styled.ul`
+  position: absolute;
+  left: 0;
+  right: 0;
+  max-height: 17.25rem;
+  overflow-y: auto;
+  background: var(--common-100);
+  border-left: 1px solid var(--neutral-90);
+  border-right: 1px solid var(--neutral-90);
+  border-bottom: 1px solid var(--neutral-90);
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  z-index: 1000;
+`;
+
+const OptionItem = styled.li`
+  padding: 1rem 1.5rem;
+  background: var(--common-100);
+  color: var(--neutral-20);
+  white-space: nowrap;
+
+  cursor: pointer;
+  transition: all 0.2s ease;
+
+  &:hover {
+    filter: brightness(0.97);
+  }
+
+  ${(props) =>
+    props.$isSelected &&
+    `
+    background: var(--cool-neutral-98);
```

Comment:
💊 이거는 정말 별 거 아니긴 한데 Dropdown2, 3에 쓰이는 색상과 헷갈리신 듯 합니다! 피그마 스타일이랑 맞게 suggestion 해 두었는데 참고해주시면 감사하겠습니다.

```suggestion
    background: var(--cool-neutral-99);
```

Reply:
수정했습니다!

Reply:
아, 이건 hover일 때와 selected일 때 색상 차이가 없어서 일부러 조금 더 어둡게 설정해둔 것이었습니다!

Reply:
👍🏻 오 그렇군요 답변 감사합니ㄷ가!! 그럼 Resolve 처리해두도록 할게요!!!